home-service/src/components/others/Avatar/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { NavigateFunction } from "react-router-dom";
+import { ROUTES } from "@/router/Routes";
+import { DropdownButton } from "../UserDropdownMenu/UserDropdownMenu";
+
+export const getUserDropdownButtons = (navigate: NavigateFunction, logout: () => void): DropdownButton[] => {
```

Comment:
šiaip šitas nelabai yra reikalingas, nes neverta iškelti ir per props paduoti, nes to UserDropDown daugiau nepanaudosi, tai siūlau tiesiog šitą sukelti į dropdowno komponentą be funkcijų